File: SDKs/googleapis.com/identitytoolkit/v2/java/lib/src/main/java/openapisdk/models/shared/GoogleCloudIdentitytoolkitAdminV2MonitoringConfig.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * GoogleCloudIdentitytoolkitAdminV2MonitoringConfig
 * Configuration related to monitoring project activity.
**/
public class GoogleCloudIdentitytoolkitAdminV2MonitoringConfig {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("requestLogging")
    public GoogleCloudIdentitytoolkitAdminV2RequestLogging requestLogging;
    public GoogleCloudIdentitytoolkitAdminV2MonitoringConfig withRequestLogging(GoogleCloudIdentitytoolkitAdminV2RequestLogging requestLogging) {
        this.requestLogging = requestLogging;
        return this;
    }
}

File: SDKs/googleapis.com/identitytoolkit/v2/python/src/sdk/models/shared/googlecloudidentitytoolkitadminv2monitoringconfig.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import googlecloudidentitytoolkitadminv2requestlogging as shared_googlecloudidentitytoolkitadminv2requestlogging


@dataclass_json
@dataclasses.dataclass
class GoogleCloudIdentitytoolkitAdminV2MonitoringConfig:
    r"""GoogleCloudIdentitytoolkitAdminV2MonitoringConfig
    Configuration related to monitoring project activity.
    """
    
    request_logging: Optional[shared_googlecloudidentitytoolkitadminv2requestlogging.GoogleCloudIdentitytoolkitAdminV2RequestLogging] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('requestLogging') }})
    
